src/imageproxy.rs

```diff
@@ -415,6 +415,18 @@ impl ImageProxy {
         Ok((fd, finish))
     }
 
+    ///Exposed LayerInfoForCopy 
+    #[instrument]
+    pub async fn layer_info_json(
+        &self,
+        img: &OpenedImage
+    ) -> Result<Vec<u8>> {
+        tracing::debug!("Getting layer info");
+        let reply = self.impl_request("LayerInfoJSON", [img.0]).await?;
+        let layers: Vec<u8> = reply.0;
```

Comment:
I think instead of a `Vec<u8>` here we want to define a structure for the value here, and return that.